internal/backend/remote-state/pg/backend.go

```diff
@@ -96,7 +107,42 @@ func (b *Backend) configure(ctx context.Context) error {
 	data := b.configData
 
 	b.connStr = data.Get("conn_str").(string)
-	b.schemaName = pq.QuoteIdentifier(data.Get("schema_name").(string))
+
+	schemaName, ok := data.Get("schema_name").(string)
```

Comment:
Why did we move quoting from here to every other SQL query?

Reply:
Keeping track of variables that may or may not be quoted become hard to track and in some instances I needed the unquoted value. Originally I started a parallel set of variables, but it looked odd to pass in the same variable with a quoted and non quoted version. 

Moving it closer to sql query made it much easier to understand. The only drawback being we might have to quote the same variable more then once.

Reply:
I am going to close this for now. Feel free to re-open.